examples/basic/main.c

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) The mldsa-native project authors
+ * SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
+ */
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/* Import public mldsa-native API
+ *
+ * This requires specifying the parameter set and namespace prefix
+ * used for the build.
+ */
+#include "../../mldsa/common.h"
+#include "../../mldsa/fips202/fips202.h"
+#include "../../mldsa/sign.h"
```

Comment:
A consumer should not be including these headers. There should be a mldsa_native.h and that should be all that is included. I think we can live with including `sign.h` for now until https://github.com/pq-code-package/mldsa-native/issues/354 is resolved. 

common.h is probably not needed. 
fips202.h can definitely not be included. These are not public APIs.

Reply:
I needed fips202.h to hash the signature... what do you suggest?

Reply:
The only thing I can think of is adding another SHA-3 implementation, but that is ugly. Sorry, using the hash was not a good idea - my bad. Maybe we switch back to comparing the actual signature, but include it from another file?

Other suggestions are welcome.

Reply:
I like keeping the full signature, as I imagine a customer use-case being swaping out the example message and getting to generate full signatures on their own messages. I'll keep the full signature check and include the extra file.